package.json

```diff
@@ -29,6 +29,8 @@
   "devDependencies": {
     "benchmark": "^1.0.0",
     "bluebird": "^2.9.12",
+    "eslint": "^2.7.0",
+    "eslint-config-loopback": "^1.0.0",
```

Comment:
You are missing `"lint": "eslint ."` and `"posttest": "npm run lint"` in this file. Also please check for the other PRs while you're at it too because I forgot to mention it in those ones.